src/components/cells/cell-output.jsx

```diff
@@ -110,28 +110,36 @@ const arrayHandler = {
   render: (value) => {
     const dataSetInfo = `${value.length} element array`
     const len = value.length
-
-    let arrayElements = _.range(len > 200 ? 100 : len - 2).map(i => (
-      <span key={`array_elt_${i}`} title={`array index: ${i}`}>
-        {renderValue(value[i], true)}{', '}
-      </span>
-    ))
-    if (len > 200) {
-      arrayElements.push(<span key="array_elts omitted">…, </span>)
-      arrayElements = arrayElements
-        .concat(_.range(len - 100, len - 2)
-          .map(i => (
-            <span key={`array_elt_${i}`} title={`array index: ${i}`}>
-              {renderValue(value[i], true)}{', '}
-            </span>
-          )))
+    let arrayElements
+    if (len > 0) {
+      arrayElements = _.range(len > 200 ? 100 : len - 1).map(i => (
```

Comment:
i don't believe this range adjustment is correct

Reply:
It been done on line 131